### ISSUE TITLE
Different editing strategies for top level data libraries vs contents.

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#8188]
Top level folder:

![image](https://user-images.githubusercontent.com/458683/59754909-55d87480-9287-11e9-8cbb-5e9f22c54a9d.png)

subfolder:

![image](https://user-images.githubusercontent.com/458683/59754886-4b1ddf80-9287-11e9-9902-f69e742eae98.png)

I recognise it is "library vs folder", but it could be more unified, I guess most people just see "folders" and not this library distinction.

**Comment by martenson** [comment]
The missing buttons in the top screen are certainly a bug. The other differences are mostly due to folders offering batch/select interface and libraries being single operation. How would you envision it?

**Comment by hexylena** [comment]
Everything like folders? Just that they're completely consistent between the two, buttons, editing, delete button position, etc?

**Comment by martenson** [comment]
@erasche they offer different options though, and e.g. batch delete on libraries is something I feel is not needed in the UI. 

However I am certainly open to redesign discussion.

**Comment by hexylena** [comment]
Yeah, I can understand not *needing* it, was just more a feeling of "eh? Why are these different / why do I have to find the buttons in different places." 

I know I've also filed a lot of bugs lately, so feel free to disregard as not important to fix
